src/services/profiles/groups.ts

```diff
@@ -7,18 +8,26 @@ export default (
   httpAuth: HttpInstance
 ): ProfilesGroupsService => ({
   async create(profileId, requestBody, options) {
+    const customProperties = ['custom_fields'];
+
     return (
-      await client.post(httpAuth, `/${profileId}/groups`, requestBody, options)
+      await client.post(
+        httpAuth,
+        `/${profileId}/groups`,
+        requestBody,
+        addCustomPropertiesToConfig(customProperties, httpAuth, options)
```

Comment:
(no need to change, just a thought) I think this is good, but I do feel like it could be as easy as `{ ...options, customFields }` and "hide" the complexity of the transformation the custom fields to the relevant "transform..." functions.

But that might be a step too far. Was trying to simplify this in my head, as this piece of code is going to end up in a lot of places